src/Instrumentation/Symfony/README.md

```diff
@@ -49,14 +38,3 @@ From Symfony subdirectory:
 $ composer install
 $ ./vendor/bin/phpunit tests
 ```
-
-## Configuration
-
-Parts of this auto-instrumentation library can be configured, more options are available throught the
-[General SDK Configuration](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration):
-
-| Name                                | Default value | Values                  | Example | Description                                                                     |
-|-------------------------------------|---------------|-------------------------|---------|---------------------------------------------------------------------------------|
-| OTEL_PHP_DISABLED_INSTRUMENTATIONS  | []            | Instrumentation name(s) | symfony | Disable one or more installed auto-instrumentations, names are comma seperated. |
```

Comment:
Is it worth keeping a reference to the instrumentation name somewhere in this document? Or adding it to the referenced instructions?

Reply:
good idea, updated to mention config.